src/Analyzers/CSharp/Tests/UseIsNullCheck/UseIsNullCheckForReferenceEqualsTests.cs

```diff
@@ -17,7 +17,9 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseIsNullCheck
     public partial class UseIsNullCheckForReferenceEqualsTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
     {
         private static readonly ParseOptions CSharp7 = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7);
-        private static readonly ParseOptions CSharp9 = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp8);
+
+        // Switch this to actually reference LanguageVersion.CSharp9 when available.
```

Comment:
do we have a tracking issue for that availability?

Reply:
There's an email thread going on right now about this.  I can open a tracking issue though to make sure htis doesn't get lost.

Reply:
https://github.com/dotnet/roslyn/issues/43348.  Thanks!